docs/features/specification.md

```diff
@@ -11,6 +11,11 @@ spec:
   # Number of desired pods.
   # Defaults to 1.
   replicas: 5
+  analysis:
+    # limits the number of failed analysis runs to be stored in a history
+    succeedRunHistoryLimit: 0
+    # limits the number of succeeded analysis runs to be stored in a history
+    failedRunHistoryLimit: 0
```

Comment:
1. We need to document the default behavior when these are omitted
2. We should document that this covers both AnalysisRuns and Experiments
3. It seems you mixed up the descriptions of successful vs. failed
4. Does failedRunHistoryLimit also cover Errors? If so should we name this to `unsuccessfulRunHistoryLimit`?

Reply:
1. Defaults. Done
2. TODO
3. Done
4. I fixed this. Now it covers the failed only.

Reply:
@jessesuen This feature is covering the AnalysisRuns only.

Maybe we can rename this ticket for AnalysisRuns only, and create another one for the experiments?
For the experiments we need to create other configuration properties I think `analysis.successfulLimit` it's only for analysis

Reply:
1. Done
2. Documented these params covers both AnalysisRuns and Experiments. Done
3. Done
4. Implement `unsuccessfulRunHistoryLimit`. Done